assignments/prototypes.js

```diff
@@ -33,76 +51,90 @@
   * should inherit takeDamage() from CharacterStats
 */
  
+const Humanoid = (humanCahracter) => {
+    this.team = humanCahracter.team;
+    this.weapons = humanCahracter.weapons;
+    this.language = humanCahracter.language;
+    GameObject.call(this.destroy, humanCahracter);
+    CharacterStats.call(this.takeDamage);
+}
+
```

Comment:
remember you will need to you object.create to pull in the prototype from the parent